Example/SPGDPRExampleAppUITests/SPGDPRExampleAppUITests.swift

```diff
@@ -20,59 +20,143 @@ class SPGDPRExampleAppUITests: QuickSpec {
             self.app = ExampleApp(XCUIApplication())
             self.app.launch()
             self.app.clearConsentButton.tap()
+            Nimble.AsyncDefaults.Timeout = 10
+            Nimble.AsyncDefaults.PollInterval = 1
         }
 
-        afterSuite {
+        afterEach {
```

Comment:
I believe we shouldn't run the following two lines in the `afterEach` block:
```swift
Nimble.AsyncDefaults.Timeout = 1
Nimble.AsyncDefaults.PollInterval = 0.01
```
That might be one of the reasons the tests are failing. 

Try moving them to an `afterSuit` block.